tests/framework/utils.py

```diff
@@ -528,6 +530,31 @@ def run_cmd_list_async(cmd_list):
     )
 
 
+def configure_git_safe_directory():
+    """
+    Add the root firecracker git folder to safe.directory config.
+
+    Firecracker root git folder in the container is
+    bind-mounted to a folder on the host which is mapped to a
+    user that is different from the user which runs the integ tests.
+    This difference in ownership is validated against by git.
+    https://github.blog/2022-04-12-git-security-vulnerability-announced/
+
+    :return: none
+    """
+    # devtool script will set the working directory to FC_ROOT/tests.
+    # We will need the parent for safe.directory configuration
+    working_dir = Path(os.getcwd())
+
+    try:
+        utils.run_cmd('git config --global '
+                      '--add safe.directory {}'.format(working_dir.parent))
+    except ChildProcessError as error:
+        assert False, "Failure to set the safe.directory " \
+                      "git config to [{}] required for gitlint tests: " \
+                      "{}".format(working_dir.parent, error)
```

Comment:
```suggestion
        raise Exception("Failure to set...") from error
```
This allows chaining the errors for giving more infomation, see https://docs.python.org/3/tutorial/errors.html#exception-chaining.